controllers/glance_controller.go

```diff
@@ -588,12 +532,19 @@ func (r *GlanceReconciler) reconcileNormal(ctx context.Context, instance *glance
 	instance.Status.Conditions.MarkTrue(condition.InputReadyCondition, condition.InputReadyMessage)
 	// run check OpenStack secret - end
 
+	db, result, err := r.ensureDB(ctx, helper, instance)
```

Comment:
@stuggi not sure if there's a reason to not overwrite the same `ctrlResult` variable here and use `result`

Reply:
no, not really a reason.

Reply:
changed to the ctrlResult